website/search_migration/migrate.py

```diff
@@ -73,6 +77,9 @@ def migrate(delete, index=None, app=None):
 
     set_up_alias(index, new_index)
 
+    # migrate nodes modified since start
+    migrate_nodes(new_index, query=Q('date_modified', 'gte', start_time))
```

Comment:
This is to catch any nodes that were modified while `migrate_nodes` was running.

1. Start the clock
2. Do all the institutions
3. Do all the nodes
4. Do all the users
5. Do any nodes that have been modified since the clock started.

If I removed the other one anything modified outside of the running time wouldn't be migrated to ES.